test/test_torch.py

```diff
@@ -3851,6 +3851,8 @@ def test_serialization(self):
         for use_name in (False, True):
             with tempfile.NamedTemporaryFile() as f:
                 handle = f if not use_name else f.name
+                if sys.platform == 'win32' and use_name:
+                    handle = tempfile.mktemp()
```

Comment:
I'm confused now. We have already created a temporary file, so why do this?

Reply:
Same with above.

Reply:
I guessed that, but why? You seem to be able to correctly retrieve `f.name` above, so why doesn't it work? BTW `mktemp()` is deprecated since Python 2.3 in favor of `mkstemp()`

Reply:
According to the web page [here](https://bugs.python.org/issue14243), it's required on Windows to close the file before reading it. Therefore the delete flag must be specified.